randomizer.py

```diff
@@ -133,6 +134,14 @@ def secondEdition(self, cards):
             self._secondEdition = CardList()
         self._AddCards(self._secondEdition, cards)
 
+    @property
+    def actionCards(self):
```

Comment:
The other relevant properties are not suffixed with "cards," and neither should this. This should just be `actions`.